script.js

```diff
@@ -0,0 +1,524 @@
+                case 'langKey': {
+                    keyElement.addEventListener('click', () => {
+                        changeLanguage();
+                    });
+                    break;
+                }
+                case 'Backspace': {
+                    keyElement.addEventListener('click', () => {
+                        value = value.substring(0, value.length - 1);
+                        onInput();
+                    });
+                    break;
+                }
+                case 'Tab': {
+                    keyElement.addEventListener('click', () => {
+                        value = value + '   ';
+                        onInput();
+                    });
+                    break;
+                }
+                case 'Space': {
+                    keyElement.addEventListener('click', () => {
+                        value = value + ' ';
+                        onInput();
+                    });
+                    break;
+                }
+                case 'ShiftLeft': {
+                    keyElement.addEventListener('mousedown', () => {
+                        changeCase();
+                    });
+                    keyElement.addEventListener('mouseup', () => {
+                        changeCase();
+                    });
+                    break;
+                }
+                case 'ShiftRight': {
+                    keyElement.addEventListener('mousedown', () => {
+                        changeCase();
+                    });
+                    keyElement.addEventListener('mouseup', () => {
+                        changeCase();
+                    });
+                    break;
+                }
+                case 'CapsLock': {
+                    keyElement.addEventListener('click', () => {
+                        changeCase();
+                        if (capsLock === 'lowerCase') {
+                            document.querySelector('.CapsLock').classList.remove('keyboard__key_inner');
+                        } else if (capsLock === 'upperCase') {
+                            document.querySelector('.CapsLock').classList.add('keyboard__key_inner');
+                        }
+                    });
+                    break;
+                }
+                case 'Enter': {
+                    keyElement.addEventListener('click', () => {
+                        value = value + '\n';
+                        onInput();
+                    });
+                    break;
+                }
+                case 'AltLeft': {
+                    onInput();
+                    break;
+                }
+                case 'AltRight': {
+                    onInput();
+                    break;
+                }
+                case 'ControlLeft': {
+                    onInput();
+                    break;
+                }
+                case 'ControlRight': {
+                    onInput();
+                    break;
```

Comment:
The same 
`case 'AltLeft':
case 'AltRight':
case 'ControlLeft':
 case 'ControlRight': {
                    onInput();
                    break;
}`